src/modules/vtol_att_control/tiltrotor.h

```diff
@@ -99,8 +99,7 @@ class Tiltrotor : public VtolType
 	float _tilt_control;
 	float _roll_weight_mc;
```

Comment:
Correct. There are a number of things that can be aligned with what's been done for the standard VTOL but I wanted to minimise changes here for now. I will revisit with a more extensive cleanup soon.